shared/oval/sshd_limit_user_access.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
```

Comment:
but the path `shared/templates/static/oval` does not seem to exist: 
https://github.com/OpenSCAP/scap-security-guide/tree/master/shared/templates/static